src/Azure.Functions.ArtifactAssembler/ArtifactAssembler.cs

```diff
@@ -137,8 +162,9 @@ private static string GetCoreToolsProductVersion(string artifactDirectoryName)
 
         private async Task CreateVisualStudioCoreToolsAsync()
         {
+            Console.WriteLine($"Starting to assemble Visual Studio Core Tools artifacts");
```

Comment:
You only need the "$" for string interpolation

```suggestion
            Console.WriteLine("Starting to assemble Visual Studio Core Tools artifacts");
```